spec/features/admin_content_management_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.feature "AdminContentManagement", type: :feature do
+  it "should be able to add new content" do
+    visit "admin/contents"
+    fill_in "content[title]", with: "This is a 10+ char descriptive title"
+    fill_in "content[body]", with: "My body is ready."
+    click_button "Submit"
+
+    expect(page).to have_content("Content successfully created.")
+    expect(current_path).to eql(admin_contents_path)
+  end
+
+  it "should not be able to add content with a 10+ char title" do
+    visit "admin/contents"
+    fill_in "content[title]", with: "Too short"
+    fill_in "content[body]", with: "My body is ready."
+    click_button "Submit"
+
+    expect(page).to have_content("The min title length is 10 characters.")
+  end
+
+  it "should not be able to add content with a 10+ char title" do
+    visit "admin/contents"
+    fill_in "content[title]", with: "60+characterslong60+characterslong60+characterslong60+characterslong"
```

Comment:
To make the character count more easily distinguishable, you might try "a" * 61 so that you get a 61 character string.